Core.UnitTests/ReadInput/InputReaderTests.cs

```diff
@@ -28,6 +28,50 @@ namespace Remotion.ReleaseProcessAutomation.UnitTests.ReadInput;
 [TestFixture]
 internal class InputReaderTests
 {
+  [Test]
+  [TestCase("y", true)]
+  [TestCase("Y", true)]
+  [TestCase("n", false)]
+  [TestCase("N", false)]
+  public void ReadConfirmation_WithInput_ReturnsOutput (string inputText, bool output)
+  {
+    var testConsole = new TestConsole();
+    testConsole.Input.PushTextWithEnter(inputText);
+    var inputReader = new InputReader(testConsole);
+
+    var act = inputReader.ReadConfirmation();
+
+    Assert.That(act, Is.EqualTo(output));
+  }
+
+  [Test]
+  public void ReadConfirmation_WithImproperInput_RepeatsInput ()
+  {
+    var testConsole = new TestConsole();
+    testConsole.Input.PushTextWithEnter("notACorrectInput");
+    testConsole.Input.PushTextWithEnter("y");
+    var inputReader = new InputReader(testConsole);
+
+    var act = inputReader.ReadConfirmation();
+
+    Assert.That(testConsole.Output, Does.Contain("The input 'notACorrectInput' is not a valid option."));
+    Assert.That(act, Is.EqualTo(true));
+  }
+
+  [Test]
+  [TestCase(true)]
+  [TestCase(false)]
+  public void ReadConfirmation_WithEnterInput_ReturnsDefault (bool defaultVal)
+  {
+    var testConsole = new TestConsole();
+    testConsole.Input.PushKey(ConsoleKey.Enter);
+    var inputReader = new InputReader(testConsole);
+
+    var act = inputReader.ReadConfirmation(defaultVal);
+
+    Assert.That(act, Is.EqualTo(defaultVal));
+  }
+
```

Comment:
Is it possible to test ReadConfirmation in a unit test? if so, please do.

Reply:
I don't think I understand, these are already the unit tests for ReadConfirmation. Did you mean the values that are input and output to and from the console?

Reply:
As discussed, the message is not tested as of yet, the rest is already tested. I didn't realize the scope of the tests that exist in the PR.